trusted/trts_sys/build.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2022 The MobileCoin Foundation
+
+//! Builds the FFI function bindings for trts (trusted runtime system) of the
+//! Intel SGX SDK
+extern crate bindgen;
+use cargo_emit::{rustc_link_lib, rustc_link_search};
+use std::{env, path::PathBuf};
+
+static DEFAULT_SGX_SDK_PATH: &str = "/opt/intel/sgxsdk";
+
+#[cfg(feature = "hw")]
+const SGX_SUFFIX: &str = "";
+#[cfg(not(feature = "hw"))]
+const SGX_SUFFIX: &str = "_sim";
+
+fn sgx_library_path() -> String {
+    env::var("SGX_SDK").unwrap_or_else(|_| DEFAULT_SGX_SDK_PATH.into())
+}
```

Comment:
Since this bit repeats across build scripts, how about we move this logic into a shared helper?